tests/test_dict.py

```diff
@@ -6,24 +8,24 @@
 class TestSharedMemoryDict:
     @pytest.fixture
     def shared_memory_dict(self):
-        smd = SharedMemoryDict(name='ut', size=1024)
+        smd = SharedMemoryDict(name="ut", size=1024)
```

Comment:
could you revert this change to use a single quote? we use this pattern throughout the project

Reply:
I think we should configure black to avoid that change. I'll do it in another PR.